saleor/page/migrations/0015_migrate_from_draftjs_to_editorjs_format.py

```diff
@@ -24,7 +24,7 @@ def parse_to_editorjs(data):
     list_data = {}
     for block in blocks:
         # if block doesn't have key it means it isn't in draft.js format
-        if not block.get("key"):
+        if block.get("key") is None:
```

Comment:
Wouldn't it be the safest to check if `key` is present in block: `if "key" in block`?